packages/happy-dom/src/fetch/ResourceFetchHandler.ts

```diff
@@ -34,7 +34,14 @@ export default class ResourceFetchHandler {
 		// We want to only load SyncRequest when it is needed to improve performance and not have direct dependencies to server side packages.
 		const absoluteURL = RelativeURL.getAbsoluteURL(document.defaultView.location, url);
 		const syncRequest = require('sync-request');
-		const response = syncRequest('GET', absoluteURL);
+		const response = syncRequest('GET', absoluteURL, {
+			headers: {
+				'user-agent': document.defaultView.navigator.userAgent,
+				cookie: document.defaultView.document.cookie,
+				referer: document.defaultView.location.href,
+				pragma: 'no-cache'
```

Comment:
```suggestion
				referer: document.defaultView.location.href
```

I think "pragma: 'no-cache'" is added in the Chrome and FF dev tools to tell the server to not cache, but it should probably not be used by default.